src/ScheduledUpgradeable.sol

```diff
@@ -190,6 +190,22 @@ abstract contract ScheduledUpgradeable is IScheduledUpgradeable, Initializable,
         return (message, messageAsG1Bytes);
     }
 
+    /// @notice Converts minimum contract upgrade delay parameters to a BLS G1 point and its byte representation.
+    /// @param _minimumContractUpgradeDelay The new minimum delay in seconds
+    /// @param nonce The nonce for the update request
+    /// @return message The original encoded message
+    /// @return messageAsG1Bytes The byte representation of the BLS G1 point
+    function minimumContractUpgradeDelayParamsToBytes(uint256 _minimumContractUpgradeDelay, uint256 nonce)
+        public
+        view
+        virtual
+        returns (bytes memory, bytes memory)
+    {
+        bytes memory message = abi.encode(_minimumContractUpgradeDelay, nonce, getChainId());
```

Comment:
I think we need some extra params here to make it secure - signing two uint256s seems like it could collide with some other dcipher signatures.
I'd suggest:
- some string relating to contract upgrade (e.g. "change-upgrade-delay")
- the address of the contract being upgraded (otherwise you can reuse signatures to change the upgrade delay on other contracts on the same chain)